bindings/python/tests/CMakeLists.txt

```diff
@@ -4,4 +4,22 @@
 # This software may be modified and distributed under the terms of the
 # BSD-3-Clause license. See the accompanying LICENSE file for details.
 
-# TODO
+macro(add_python_unit_test pythonscript)
+  find_program(PYTHON_EXECUTABLE
```

Comment:
`PYTHON_EXECUTABLE` is already defined in parent scope, also respecting the `YARP_USE_PYTHON_VERSION`, so I would avoid the `find_program` here (note tha this is different from Lua because the FindLua.cmake module in CMake does not search for the lua interpreter, so we need to search it autonomously).